src/lib/containers/works_for_you.tsx

```diff
@@ -128,6 +131,10 @@ export class WorksForYou extends React.Component<Props, State> {
     })
   }
 
+  componentDidUpdate() {
+    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
```

Comment:
Ok it doesn’t break, but it also doesn’t produce a great result. Maybe `this.currentScrollOffset` should be set to `0` in the `constructor`, just to be safe? (Then you also don’t need to mark `currentScrollOffset` as optional.

```
$ node
> undefined + 1
NaN
```